css/base.css

```diff
@@ -0,0 +1,55 @@
+:root {
+  --primary-color: #008cd6;
+  --primary-dark-color: #0069a1;
+  --primary-light-color: #99d1ef;
+  --secondary-color: #1a1a66;
+  --background-color: #eeeeee;
+  --text-primary: #111111;
+  --text-black-light: rgba(0, 0, 0, 0.65);
+  --text-black-hover: rgba(0, 0, 0, 0.85);
+  --text-hover-background: rgba(121, 82, 179, 0.1);
+  --logo-font-family: "Mochiy Pop P One", sans-serif;
+  --primary-font-family: "Montserrat", sans-serif;
+  --seconary-font-family: "Roboto Mono", monospace;
+  --font-size-sm: 1.2rem;
+  --font-size-lg: 1.4rem;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: var(--background-color);
+  font-family: var(--seconary-font-family);
+  line-height: 1.5;
+  overflow-x: hidden;
+}
+
+.visually-hidden {
```

Comment:
can move this to utility CSS. I guess we should restrict ourselves to resetting browser style to all elements.

Reply:
can keep here